docs/sphinx/reference-libobs-util-dstr.rst

```diff
@@ -87,8 +87,29 @@ General String Helper Functions
 
 .. function:: char **strlist_split(const char *str, char split_ch, bool include_empty)
 
-   Splits a string in to a list of multiple sub-strings.  Free with
-   :c:func:`strlist_free()`.
+   Splits a string in to a list of multiple sub-strings, terminated by
+   ``NULL``. If ``split_ch`` does not exist in the string, the first
+   sub-string will be the same as ``str``. Free with :c:func:`strlist_free()`.
+
```

Comment:
Please document the other two parameters using the `:param:` format too.

Reply:
Updated, please check